python-listener/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 EXPOSE 80
 
 # Run main.py when the container launches
```

Comment:
Adjust comment to new python filename

Reply:
Also need to adjust paas.mk to reflect new directory (python-listener) and new python file to run (app.py)